### PR TITLE
[terra-clinical-data-grid] Add theme context

### PR DESCRIPTION
**Pull request by saket2403**
### Summary
Apply theme context to terra-clinical-data-grid

### Deployment Link

https://terra-clinical-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-clinical-data-grid/src/orion-fusion-theme/RowSelectionCell.module.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+:local {
```
Comment (emilyrohrbough) [inline packages/terra-clinical-data-grid/src/orion-fusion-theme/RowSelectionCell.module.scss:1]: @import '~terra-mixins/lib/Mixins';
```suggestion
:local {
```

File: packages/terra-clinical-data-grid/src/clinical-lowlight-theme/RowSelectionCell.module.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+:local {
```
Comment (emilyrohrbough) [inline packages/terra-clinical-data-grid/src/clinical-lowlight-theme/RowSelectionCell.module.scss:1]:
```suggestion
@import '~terra-mixins/lib/Mixins';

:local {
```

File: packages/terra-clinical-data-grid/src/orion-fusion-theme/RowSelectionCell.module.scss (hunk 1 above)
Comment (saket2403) [reply packages/terra-clinical-data-grid/src/orion-fusion-theme/RowSelectionCell.module.scss:3]: Added 46e8de9

File: packages/terra-clinical-data-grid/src/clinical-lowlight-theme/RowSelectionCell.module.scss (hunk 2 above)
Comment (saket2403) [reply packages/terra-clinical-data-grid/src/clinical-lowlight-theme/RowSelectionCell.module.scss:3]: Added 46e8de9

**Comment by saket2403** [comment]
> Did we add the theme context to RowSelectionCell.jsx?

Added 46e8de9

File: packages/terra-clinical-data-grid/src/subcomponents/Row.jsx [hunk 3]
```diff
@@ -65,12 +67,22 @@ class Row extends React.Component {
...
       ...customProps
     } = this.props;
+    const theme = this.context;
+
+    const rowClass = classNames(
+      cx(
+        'row',
+        { selected: isSelected, striped: isStriped },
+        theme.className,
+        customProps.className,
```
Comment (mjhenkes) [inline packages/terra-clinical-data-grid/src/subcomponents/Row.jsx:77]: custom props should be defined outside of cx

Comment (saket2403) [reply packages/terra-clinical-data-grid/src/subcomponents/Row.jsx]: Fixed 7e45e75

**Pull request merged**